CHANGES.md

```diff
@@ -46,6 +50,7 @@ To be released.
  -  `Swarm<T>` became to send 100 blocks (instead of 500 blocks) for each
     reply during IDL, in order to stabilize connection in high latency
     environments.  [[#679]]
+ -  `BlockChain<T>` became to contain genesis-block always.  [[#688]]
```

Comment:
```suggestion
 -  `BlockChain<T>` now requires genesis-block.  [[#688]]
```

Reply:
This line was replaced with the comment in `BlockChain<T>` class.